docs/monitoring/custom-metrics.md

```diff
@@ -1,394 +1,108 @@
-
-Content-type: application/json
-
-Method: POST
-
- 
-
-JSON body:
-
-<table>
-<thead>
-<tr class="header">
-<th>Field name</th>
-<th>Field type</th>
-<th>Constraints</th>
-</tr>
-</thead>
-<tbody>
-<tr class="odd">
-<td>datapoints</td>
-<td>array of 'data point'</td>
-<td>elements count &lt;= 100</td>
-</tr>
-</tbody>
-</table>
-
- 
-
-Data point:
-
-<table>
-<thead>
-<tr class="header">
-<th>Field name</th>
-<th>Description</th>
-<th>Field type</th>
-<th>Constraints</th>
-<th>Required</th>
-</tr>
-</thead>
-<tbody>
-<tr class="odd">
-<td>timestamp</td>
-<td>Time in milliseconds since &quot;Epoch&quot;</td>
-<td>Long</td>
-<td> </td>
-<td>No</td>
-</tr>
-<tr class="even">
-<td>name</td>
-<td>Metric name</td>
-<td>String</td>
-<td>length &gt; 0 and &lt;= 255</td>
-<td>Yes</td>
-</tr>
-<tr class="odd">
-<td>value</td>
-<td>Data point value</td>
-<td>Double</td>
-<td> </td>
-<td>Yes</td>
-</tr>
-<tr class="even">
-<td>aggregation</td>
-<td>Aggregation type</td>
-<td>String</td>
-<td>&quot;min&quot;, &quot;max&quot;, &quot;avg&quot;, &quot;sum&quot;</td>
-<td>Yes</td>
-</tr>
-<tr class="odd">
-<td>filter1</td>
-<td>Value for filter1</td>
-<td>String</td>
-<td>length &gt; 0 and &lt;= 255</td>
-<td>No</td>
-</tr>
-<tr class="even">
-<td>filter2</td>
-<td>Value for filter2</td>
-<td>String</td>
-<td>length &gt; 0 and &lt;= 255</td>
-<td>No</td>
-</tr>
-</tbody>
-</table>
-
- 
-
-Example:
-
-``` JSON
-{
-   "datapoints" : [
-      {
-         "timestamp" : 1369669889927,
-         "name" : "registered-users-count",
-         "value" : 1,
-         "aggregation" : "sum",
-         "filter1" : "user.gender=male",
-         "filter2" : "account.type=free"
-      },
-      {
-         "name" : "registered-users-count",
-         "value" : 1,
-         "aggregation" : "sum",
-         "filter1" : "user.gender=female",
-         "filter2" : "account.type=paid"
-      }
- ]
-}
+appStats,token=00000000-1111-2222-3333-444444444444,os.host=host001,region=us user.requests.count=10i,user.requests.time=327i 1628605794318000000
 ```
 
-To send metrics using curl, save example above to a file named
-datapoints.json and submit it with the following call, using your own
-token:
-
-``` bash
-curl -v -H 'Content-type: application/json' -d @datapoints.json http://spm-receiver.sematext.com/receiver/custom/receive.json?token=[spm app token]
+In this example we can see:
+- `appStats` - a `measurement` or a `namespace`, it describes the type of metrics in this particular line and it can be
+  anything that makes sense to you. It will be used as a prefix to all metrics in this line (so, full name of a metric
+  `user.requests.count` is actually `appStats.user.requests.count`)
+- `token` - a token of your Sematext monitoring App
+- `os.host` and `region` - two tags used to describe these metrics. You can append more tags by appending them after
+  `region=us` (tags have to be separated by a comma)
```

Comment:
This region is unrelated to SC US vs. SC EU?  If so, maybe we could use another tag instead of 'region' to avoid potential confusion.

Reply:
Yep, I'll change it to... something else.